apps/render_cf.py

```diff
@@ -242,11 +237,50 @@ def validate_job_spec(job_type: str, job_spec: dict) -> None:
                 f'but should have {expected_param_fields}'
             )
 
+    for profile in job_spec['cost_profiles'].values():
+        if profile.keys() not in ({'cost_parameters', 'cost_table'}, {'cost'}):
+            raise ValueError(f'Cost definition for job type {job_type} has invalid keys: {profile.keys()}')
+
+        if 'cost_table' in profile:
+            if not isinstance(profile['cost_parameters'], list):
+                raise ValueError(
+                    f'Cost definition for job type {job_type} has invalid cost_parameters: Must be a list of strings.'
+                )
+
+            validate_cost_table(profile['cost_table'], job_type)
```

Comment:
We *could* update `test_validate_job_spec` in `test_render_cf.py`, but I'm not super concerned.

Reply:
Good idea, I'll add em